src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -71,7 +102,8 @@ public static ObservableCollection<PythonEngineVersion> AvailableEngines
         {
             get
             {
-                return new ObservableCollection<PythonEngineVersion>(PythonEngineManager.Instance.AvailableEngines.Select(x=> x.Version));
+                return new ObservableCollection<PythonEngineVersion>(PythonEngineManager.Instance.AvailableEngines.
```

Comment:
this weirdness again, returning a new observable, I think we may have talked about it in another PR, I know this is not new code, does it matter here?

Reply:
I figured it is acceptable...since the property is Obsolete and not very useful to access from here.